python/taichi/lang/__init__.py

```diff
@@ -34,6 +35,7 @@
 core = _ti_core
 
 runtime = impl.get_runtime()
+kernel_profiler = get_default_kernel_profiler()
```

Comment:
Do not expose this for now?

Reply:
OK, previously this was done to access `KernelProfiler` via `ti.kernel_profiler`.